README.md: put chroot style information together, extend (#32446)
void-linux/void-packages: README.md

- install(1) - GNU coreutils
- objcopy(1), objdump(1), strip(1): binutils

> `xbps-src` requires [a utility to chroot](#chroot-methods) and bind mount existing directories

Reviewer: "a chroot-like utility" maybe? or `chroot(1)`

Author: sentence reads "utility to chroot and bind mount existing directories", hard to rewrite

#### xbps-uchroot(1)

This is the only method that implements functionality of `xbps-src -t`, therefore the

> flag enables it ignoring choice in configuration file.

Reviewer: ```suggestion
flag enables it and ignores the choice made in configuration files.
```

Author: done

> XBPS utility that uses `namespaces` and must be `setgid` (part of xbps).
> 

Reviewer: ```suggestion
XBPS utility that uses `namespaces` and must be `setgid` (part of `xbps` package in Void Linux).

```

Maybe this should go before the "This is the only..." paragraph?

Author: It is part of xbps whereever it is extracted, remark is more useful when you are outside Void.

your user is a member of the required `group` and that `xbps-uchroot(1)` utility has the
proper permissions and owner/group as explained above.

#### bwrap(1)

bubblewrap, setuid implementation of a subset of user namespaces
for unprivileged users. Requires no other permissions.

> See <https://github.com/containers/bubblewrap>.

Reviewer: Our bwrap isn't suid, it uses user namespaces as well. I think something like "sandbox tool that can use user namespaces or be suid" would be more adequate.

Author: done

    $ tar xvf xbps-static-latest.<arch>-musl.tar.xz -C ~/XBPS
    $ export PATH=~/XBPS/usr/bin:$PATH

> If xbps-uunshare does not work because of no `user namespaces` support,
> try other [chroot methods](#chroot-methods).

Reviewer: ```suggestion
If `xbps-uunshare` does not work because of lack of `user_namespaces(7)` support,
try other [chroot methods](#chroot-methods).
```

Author: done
